Modules/_hashopenssl.c

```diff
@@ -800,6 +913,33 @@ EVPnew(PyObject *module, const EVP_MD *digest,
     return (PyObject *)self;
 }
 
+static PyObject*
+EVP_raw_new(PyObject *module, PyObject *data_obj, const char *name,
```

Comment:
it's surprising to see OpenSSL "EVP_" prefix, whereas the function is a "Python" function. Would it be possible to use a different name to avoid confusion? Maybe "PY_EVP_raw_new" or "_Py_EVP_raw_new" or anything else.

Reply:
all EVP object functions have the ``EVP_`` prefix. I don't want to break with the existing coding convention. It's unrelated to the patch.

Reply:
I'd go ahead and start using better names for the internal functions as an incremental improvement. `_Py_EVP_raw_new` makes sense.

Reply:
I got rid of ``EVPnew`` and changed the name of the function to ``py_evp_fromname``.